src/components/Search/Search.test.tsx

```diff
@@ -246,7 +246,7 @@ describe('<Search /> component test', () => {
       const getDetailPageURL = jest.fn(() => 'detail/page/url');
       jest.doMock(URL_FILE_PATH, () => ({ getDetailPageURL }));
 
-      const suggestionValue = [];
+      const suggestionValue: unknown[] = [];
```

Comment:
I would prefer to create an interface for the suggestionValue, but since I am refactoring the AutoComplete component we can leave it like that